client/my-sites/domains/style.scss

```diff
@@ -10,3 +11,45 @@
 		align-items: center;
 	}
 }
+
+body.is-section-domains.is-experiment-user {
```

Comment:
Do we need the `is-section-domains` class here? Doesn't this stylesheet (`/my-sites/domains/style.scss`) imply that we're already in the domains section?

Reply:
I see the same pattern being used in other places [like the /plans page](https://github.com/Automattic/wp-calypso/blob/update/implement-domains-page-for-experimentV2/client/my-sites/plans/header.scss#L3). In this particular case, we if remove it, the `is-experiment-user` will be enough to filter it, should we be less restrictive in this CSS?

Reply:
Committed code doesn't always stand the standards and best practices. Plus, there are always bits to improve. [It's a marathon, not a sprint](https://automattic.com/creed/marathon-sprint/), after all.

I'd say, if `is-experiment-use` is enough, then no need to add another class name.